docs/contents/releases/1-18/4/index.md

```diff
@@ -0,0 +1,657 @@
+# EKS-D v1-18-eks-4 Release
+
+The [EKS-D v1-18-eks-4](https://distro.eks.amazonaws.com/kubernetes-1-18/kubernetes-1-18-eks-4.yaml)
+release manifest defines the release used in EKS-D.
+
+```yaml
+---
+apiVersion: distro.eks.amazonaws.com/v1alpha1
```

Comment:
@TerryHowe I know I ask you this every single time and it's always been "no", but do we need to change the shasums on at least some of these? I'm asking specifically this time because we changed the Kubernetes patch version for a minor release, which we've never done before.